easybuild/easyconfigs/f/foss/foss-2023.09.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'Toolchain'
+
+name = 'foss'
+version = '2023.09'
+
+homepage = 'https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain'
+description = """GNU Compiler Collection (GCC) based compiler toolchain, including
+ OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK."""
+
+toolchain = SYSTEM
+
+local_gccver = '13.2.0'
+
+# toolchain used to build foss dependencies
+local_comp_mpi_tc = ('gompi', version)
+
+# we need GCC and OpenMPI as explicit dependencies instead of gompi toolchain
+# because of toolchain preparation functions
+dependencies = [
+    ('GCC', local_gccver),
+    ('OpenMPI', '4.1.5', '', ('GCC', local_gccver)),
```

Comment:
should be bumped to `4.1.6`, see also #18898

Reply:
Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/18886/commits/66b603f8f3b4765482d184672d223bbb6057352a